modules/mobile/spec/requests/mobile/v0/efolder_spec.rb

```diff
@@ -0,0 +1,29 @@
+# frozen_string_literal: true
+
+require_relative '../../../support/helpers/rails_helper'
+require 'support/stub_efolder_documents'
```

Comment:
no VCRs are available and I was having trouble recording them myself. The other efolder specs stub the response so I'm following suit

Reply:
Without vcrs is there a way we can add error case tests?